docs/index.rst

```diff
@@ -36,7 +38,7 @@ To disable compilation with AVX2 support regardless of the machine
 architecture::
 
     $ export DISABLE_NUMCODECS_AVX2=
-    $ pip install -v --no-cache-dir numcodecs
+    $ pip install -v --no-cache-dir --no-binary :all: numcodecs
```

Comment:
```suggestion
    $ pip install -v --no-cache-dir --no-binary numcodecs numcodecs
```
to install from source only numcodecs, not all dependecise (like numpy which need FORTRAN compiler)